packages/ShowMore/src/ShowMore.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import PropTypes from "prop-types";
+import truncate from "lodash/truncate";
+import uuid from "uuid/v4";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Button from "@paprika/button";
+import { toggleStyles } from "./ShowMore.styles";
+
+const propTypes = {
+  /** Additional description for "Show more" link. Should be a "topic" that will be appended to "Show more about [topic]". */
+  a11yText: PropTypes.string,
```

Comment:
if we want to make our components accessible out of the box, should this almost be required?

Reply:
I find myself in the UX accessibility compliance tickets that I just miss out on some a11yText in components, but if it was required (or somehow strongly encouraged) then i wouldn't forget

Reply:
mmmm what if you only want to be `show more` without a topic oscar?

Reply:
Yeah, it's technically accessible without the `a11yText`, it's just _better_ if there's more context for screen readers than simply "show more".  It might be a bit too heavy-handed to make it required in this case.